eda_analysis/eda_analysis.py

```diff
@@ -75,10 +75,11 @@ def describe_num_var(dataframe, num_vars):
     
     Returns:
     --------
-    dataframe: pandas.DataFrame
-        statistical summary of the numeric variables
-    altair
-        a grid of altair plot containing all histograms
+    tuple(pandas.DataFrame, altair)
+        dataframe: pandas.DataFrame
```

Comment:
only return type should be specified. not the variable name. Remove dataframe and just keep pandas.DataFrame

Reply:
Got it. Removed. Thank you!